docs/contrib/intellij.rst

```diff
@@ -0,0 +1,17 @@
+.. _intellij:
```

Comment:
Oh, I forgot. Before the Metals and IntelliJ sections, please mention that we need to execute `sbt rebuild` so that artifacts are published and the IDEs can resolve them.